golangvend

```diff
@@ -38,7 +38,12 @@ CACHE_PATH="${PWD}/.golangvend-cache"
 
 # Given a string, print its SHA256 checksum.
 hash_string() {
-  echo "${1}" | sha256sum | cut -d' ' -f1
+  if [[ uname == "darwin"||"Darwin" ]]; then
```

Comment:
Can you check whether this works on Linux? I would expect something like ´[[ "$(uname)" == "darwin" || "$(uname)" == "Darwin" ]]`.